packages/babel-cli/src/babel/index.js

```diff
@@ -126,4 +128,51 @@ if (commander.outDir) {
   fn = require("./file");
 }
 
+if(commander.settings) {
+  let fileOptionsManager = require("babel-core").File,
+      allOptions = [];
```

Comment:
I'd pull this into its own declaration, they seem to only declare things the line before they use them